src/mainviews/create-job.tsx

```diff
@@ -528,9 +530,14 @@ export function CreateJob(props: ICreateJobProps): JSX.Element {
           <Accordion
             defaultExpanded={false}
             expanded={advancedOptionsExpanded}
-            onChange={(_: React.SyntheticEvent, expanded: boolean) =>
-              setAdvancedOptionsExpanded(expanded)
-            }
+            onChange={(e: React.SyntheticEvent, expanded: boolean) => {
+              log(
+                `create-job.advanced-options.${
+                  expanded ? 'expand' : 'collapse'
+                }`
+              );
+              setAdvancedOptionsExpanded(expanded);
+            }}
           >
             <AccordionSummary
```

Comment:
Should we capture events when the accordion is expanded or collapsed?

Reply:
The log statement above this does exactly that. Did you mean to capture something else?

Reply:
My mistake; I was looking in the wrong place here.